setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = blacken_docs
-version = 1.16.0
+version = 1.17.0
```

Comment:
This change is usually decided by the release manager depending on the changelog at release time: [see example](https://github.com/adamchainz/blacken-docs/commit/960ead214cd1184149d366c6d27ca6c369ce46b6)